static/src/views/Chat.vue

```diff
@@ -252,7 +251,12 @@ export default {
             roomId: this.roomId,
           });
         });
-        this.messages = formattedMessages;
+        if (this.offset >= this.batchSize) {
+          // We are loading old messages
+          this.messages = [...formattedMessages, ...this.messages];
```

Comment:
neat, I'd never seen array-spreading used like this.

Reply:
Yeah, I got it from the chat plugin docs:

https://github.com/antoine92190/vue-advanced-chat#important-notes

I can't imagine performance matters much here, but wanted to try out the pattern with two arrays.